Makefile

```diff
@@ -1,10 +1,13 @@
 SHELL := bash
 TMPDIR ?= /tmp
+RUNDIR ?= /tmp/travis-run.d
```

Comment:
`RUNDIR` has traditionally been `/var/tmp/travis-run.d`, right?
If this is intentional, would it make sense to use another name besides `RUNDIR`?

Reply:
@soulshake You're right, it's been `/var/tmp/travis-run.d` elsewhere.  In this case, I intentionally made it something different so that, hopefully, it's not seen as a magical sacrosanct value, and hopefully help suss out any bugs that make such an assumption.  I'm happy to set it to the same value as elsewhere, though!